src/main/java/bio/terra/cli/businessobject/Workspace.java

```diff
@@ -39,10 +39,11 @@
  */
 public class Workspace {
   private static final Logger logger = LoggerFactory.getLogger(Workspace.class);
-  private UUID uuid;
-  private String userFacingId;
+  private final UUID uuid;
+  private final String userFacingId;
   private CloudPlatform cloudPlatform;
   private String googleProjectId;
+  private String awsAccountId;
```

Comment:
We might as well include all the fields from the `AwsCloudContext` here.

Reply:
No concern with that. At this time did not have a need for more fields. so skipped.  
Keeping in mind storage / other resources, those field might help, at least for logging. 
Adding all the fields to the object, limiting user display fields